packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc

```diff
@@ -42,7 +40,7 @@ class ConnectivityPlusTizenPlugin : public flutter::Plugin {
   void RegisterObserver(
       std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> &&events) {
     EnsureConnectionHandle();
-    if (connection_set_type_changed_cb(connection_, ConnectionTypeChangedCB,
+    if (connection_set_type_changed_cb(connection_, OnConnectionTypeChanged,
```

Comment:
Why don't you use a simple lambda function instead of this?

Reply:
Oh, it makes sense.